src/app/modules/auth/components/login/login.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component } from '@angular/core';
+import { FormBuilder, Validators } from '@angular/forms';
+import { LoginFormGroup } from './models';
+import { fieldErrorsMap } from '../../../../constants';
+import { AuthService } from '../../auth.service';
+import { MatSnackBar } from '@angular/material/snack-bar';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.css']
+})
+export class LoginComponent {
+  cardTitle: string = 'Login';
+  buttonLinkTitle: string = 'Registration';
+  buttonLink: string = '/auth/registration';
+  mainButtonTitle: string = 'Sign in';
+
+  formGroup: LoginFormGroup = this.fb.group({
+    email: ['', [Validators.email, Validators.required]],
+    password: ['', Validators.required],
+  });
+
+  constructor(private fb: FormBuilder, private authService: AuthService) {
+  }
+
+  onSubmit(): void {
+    const { email, password } = this.formGroup.controls;
```

Comment:
You can receive  value`  const { email, password } = this.formGroup.value  `

Reply:
Unfortunately no because FormGroup doesn't have the field `values`.